tests/integration/test_fim/test_registry/test_ignore/test_ignore_registry.py

```diff
@@ -94,17 +81,103 @@ def test_ignore_registry(key_string, key_object, tags_to_apply, get_configuratio
+    (key, subkey_1, KEY_WOW64_32KEY, "some_value", True, {'ignore_registry_value'}),
+    (key, subkey_1, KEY_WOW64_64KEY, "some_value", True, {'ignore_registry_value'}),
+    (key, subkey_1, KEY_WOW64_64KEY, "regex_ignored_value", False, {'ignore_registry_value'}),
+    (key, subkey_1, KEY_WOW64_32KEY, "regex_ignored_value", False, {'ignore_registry_value'}),
+    (key, subkey_2, KEY_WOW64_64KEY, "regex_ignored_value", False, {'ignore_registry_value'}),
+    (key, subkey_2, KEY_WOW64_64KEY, "some_value", True, {'ignore_registry_value'}),
+    (key, subkey_1, KEY_WOW64_32KEY, ignore_value, False, {'ignore_registry_value'}),
+    (key, subkey_1, KEY_WOW64_64KEY, ignore_value, False, {'ignore_registry_value'}),
+    (key, subkey_2, KEY_WOW64_64KEY, ignore_value, False, {'ignore_registry_value'})
+])
+def test_ignore_registry_value(root_key, registry, arch, value, triggers_event, tags_to_apply,
+                         get_configuration, configure_environment, restart_syscheckd, wait_for_fim_start):
+    """
+    Check registries are ignored according to configuration.
 
-    TimeMachine.travel_to_future(timedelta(hours=13))
-
-    ignored_registry = wazuh_log_monitor.start(timeout=10, callback=callback_ignore,
-                                               accum_results=len(sub_keys),
-                                               error_message='Did not receive expected '
-                                                             '"Ignoring ... due to( sregex)?" event').result()
-    ignored_registry = set(ignored_registry)
-    for ign_reg in sub_keys:
-        try:
-            ignored_registry.remove(os.path.join(keys_strings[0], ign_reg))
-        except KeyError:
-            raise KeyError(f'{os.path.join(keys_strings[0], ign_reg)} not in {ignored_registry}')
-    assert len(ignored_registry) == 0
+    Parameters
+    ----------
+    key_string : str
+        String name of the key.
+    key_object : object
+        Object winreg of the key.
+    tags_to_apply : set
+        Run test if matches with a configuration identifier, skip otherwise.
+    """
+    check_apply_test(tags_to_apply, get_configuration['tags'])
+    scheduled = get_configuration['metadata']['fim_mode'] == 'scheduled'
+    # Open the key (this shouldn't create an alert)
+    key_h = create_registry(registry_parser[root_key], registry, arch)
+    # Create values
+    modify_registry_value(key_h, value, REG_SZ, "test_value")
+    # Go ahead in time to let syscheck perform a new scan
+    check_time_travel(scheduled, monitor=wazuh_log_monitor)
+
+    event = wazuh_log_monitor.start(timeout=global_parameters.default_timeout,
+                                callback=callback_detect_event,
+                                error_message='Did not receive expected '
+                                                '"Sending FIM event: ..." event',
+                                accum_results=2 if triggers_event else 1).result()
```

Comment:
These lines are under-indented.
```suggestion
                                    callback=callback_detect_event,
                                    error_message='Did not receive expected '
                                                  '"Sending FIM event: ..." event',
                                    accum_results=2 if triggers_event else 1).result()
```